.develop/openrpc-validator-cli.js

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
is this the result of running typescript compile?

If so, might I suggest that you include the src ts instead of the compiled result?

If this is meant to be maintained as JS, then lets ditch the export and clean this up a little.

Reply:
Noted. This is/was only in the PR as a shortcut/convenience for reproducing the document validation heuristic, and I've since made [the script just demand that this tool is installed globally](https://github.com/etclabscore/core-geth/blob/feat/openrpc-describe/.develop/test.sh#L42).